### PR TITLE
Update service-data-lineage.md

### PR DESCRIPTION
**Pull request by yaroncan**
lineage view is not available on my-workspace but only on other worksapces

### CODE REVIEW

**Comment by PRMerger7** [comment]
@yaroncan : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/service-data-lineage.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ Lineage view is an interactive canvas. You can use the mouse and touchpad to nav
 ## Considerations and limitations
 
 Lineage view isn't available on Internet Explorer. See [Supported browsers for Power BI](power-bi-browsers.md) for details.
```
Comment (paulinbar) [inline powerbi-docs/service-data-lineage.md]:
```suggestion
* Lineage view isn't available on Internet Explorer. See [Supported browsers for Power BI](power-bi-browsers.md) for details.
```

File: powerbi-docs/service-data-lineage.md [hunk 2]
```diff
@@ -86,6 +86,7 @@ Lineage view is an interactive canvas. You can use the mouse and touchpad to nav
...
+Lineage view isn't available in My Workspace.
```
Comment (paulinbar) [inline powerbi-docs/service-data-lineage.md]:
```suggestion
* Lineage view isn't available in My Workspace.
```

**Comment by paulinbar** [comment]
#sign-off

**Pull request merged**